index.html

```diff
@@ -325,6 +327,10 @@ <h2><code>PointerEvent</code> interface</h2>
                         <dd>
                             <p>Indicates if the pointer represents the <a>primary pointer</a> of this pointer type.</p>
                         </dd>
+                    <dt><dfn>deviceId</dfn></dt>
+                        <dd>
+                            <p>A unique identifier for the device that generates the pointer event. Pointer events generated from the same physical device will have the same <code>deviceId</code> for the browsing session. User agents MAY reserve a generic <code>deviceId</code> value of <code>0</code> or <code>1</code> for events generated by the primary mouse device. The <code>deviceId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified.</p>
```

Comment:
The general idea seems okay, though we'll need to discuss it in the working group. I think this is missing some important details though:
* `pointerId` already allows user agents to reuse the same pointerId for a particular pointing device so without the added context that the deviceId may be -1 initially and be set later needs to be pointed out possibly with a note about hardware support.
* This should probably point to the section in pointerId where it explains that reserved ids must still be randomized between different sessions.

Is there a point in time by which you are guaranteed to have a deviceid? E.g. are you guaranteed to have one by the pointerdown?

Reply:
Thanks for the feedback, I'll work on adding these details. 

There is no guarantee for `deviceId` on pointerdown, at least for surface devices.

Reply:
I've addressed this feedback in the definition for uniqueId in deviceProperties. PTAL